pkg/action/action_handler.go

```diff
@@ -144,6 +168,32 @@ func (h *ActionHandler) ExecuteAction(actionExecutionDTO *proto.ActionExecutionD
 	return result, nil
 }
 
+func keepAlive(tracker sdkprobe.ActionProgressTracker, stop chan struct{}) {
```

Comment:
this function currently is only useful for Move Action, and I am not sure whether it is right for all the actions (other actions may use ActionProgressTracker to report real progress). So it is subject to change in the future.

 In SDK, the discoeryClient has implemented a similar function to keep alive while waiting for the probe result.  For the actionWorker, the SDK does not implement the KeepAlive function in purpose, but implement the UpdageProgress() instead, to let the action handler to report its progress.